Rom.py

```diff
@@ -1,1339 +1,1537 @@
-        tt['ganon_fall_in_alt'] = 'You cannot defeat me until you finish your goal!'
-        tt['ganon_phase_3_alt'] = 'Got wax in\nyour ears?\nI can not die!'
-    tt['kakariko_tavern_fisherman'] = TavernMan_texts[random.randint(0, len(TavernMan_texts) - 1)]
-
-    pedestalitem = world.get_location('Master Sword Pedestal').item
-    pedestal_text = 'Some Hot Air' if pedestalitem is None else pedestalitem.pedestal_hint_text if pedestalitem.pedestal_hint_text is not None else 'Unknown Item'
-    tt['mastersword_pedestal_translated'] = pedestal_text
-    pedestal_credit_text = 'and the Hot Air' if pedestalitem is None else pedestalitem.pedestal_credit_text if pedestalitem.pedestal_credit_text is not None else 'and the Unknown Item'
-
-    etheritem = world.get_location('Ether Tablet').item
-    ether_text = 'Some Hot Air' if etheritem is None else etheritem.pedestal_hint_text if etheritem.pedestal_hint_text is not None else 'Unknown Item'
-    tt['tablet_ether_book'] = ether_text
-    bombositem = world.get_location('Bombos Tablet').item
-    bombos_text = 'Some Hot Air' if bombositem is None else bombositem.pedestal_hint_text if bombositem.pedestal_hint_text is not None else 'Unknown Item'
-    tt['tablet_bombos_book'] = bombos_text
-
-    rom.write_bytes(0xE0000, tt.getBytes())
-
-    credits = Credits()
-
-    sickkiditem = world.get_location('Sick Kid').item
-    sickkiditem_text = random.choice(SickKid_texts) if sickkiditem is None or sickkiditem.sickkid_credit_text is None else sickkiditem.sickkid_credit_text
-
-    zoraitem = world.get_location('King Zora').item
-    zoraitem_text = random.choice(Zora_texts) if zoraitem is None or zoraitem.zora_credit_text is None else zoraitem.zora_credit_text
-
-    magicshopitem = world.get_location('Potion Shop').item
-    magicshopitem_text = random.choice(MagicShop_texts) if magicshopitem is None or magicshopitem.magicshop_credit_text is None else magicshopitem.magicshop_credit_text
-
-    fluteboyitem = world.get_location('Flute Spot').item
-    fluteboyitem_text = random.choice(FluteBoy_texts) if fluteboyitem is None or fluteboyitem.fluteboy_credit_text is None else fluteboyitem.fluteboy_credit_text
-
-    credits.update_credits_line('castle', 0, random.choice(KingsReturn_texts))
-    credits.update_credits_line('sanctuary', 0, random.choice(Sanctuary_texts))
-
-    credits.update_credits_line('kakariko', 0, random.choice(Kakariko_texts).format(random.choice(Sahasrahla_names)))
-    credits.update_credits_line('desert', 0, random.choice(DesertPalace_texts))
-    credits.update_credits_line('hera', 0, random.choice(MountainTower_texts))
-    credits.update_credits_line('house', 0, random.choice(LinksHouse_texts))
-    credits.update_credits_line('zora', 0, zoraitem_text)
-    credits.update_credits_line('witch', 0, magicshopitem_text)
-    credits.update_credits_line('lumberjacks', 0, random.choice(Lumberjacks_texts))
-    credits.update_credits_line('grove', 0, fluteboyitem_te
```

Comment:
Ok. So this actually works, but might be ugly without the null byte. The real question is do we even need this? technically we are not hashing just the title anymore, but doing a hash of the whole rom (or the whole patch in the case of json output).

Consider dropping option_identitfier completely, and going back to the original format with underscores.

Reply:
dropping it